packages/notification-service/src/helpers/KnownAddressesCache.ts

```diff
@@ -0,0 +1,40 @@
+import { Database, DataSnapshot } from '@firebase/database-types'
+
+const TAG = 'helpers/KnownAddressesCache'
+
+const ROOT_KEY = 'addressesExtraInfo'
+
+const ON_VALUE_CHANGED = 'value'
+
+export default class KnownAddressesCache {
```

Comment:
It would be ideal to reuse this code from the one we have in the blockchain-api.

No need to do it here of course, but longer term I can imagine us having a `@valoraapp/utils` or `@valoraapp/core` package which we can use to import this sort of things. I think there are other similar cases across our services.

Reply:
Yep, I think the same :)